ImperatorToCK3Tests/ImperatorWorldTests/Characters/CharacterTests.cpp

```diff
@@ -440,7 +441,7 @@ TEST(ImperatorWorld_CharacterTests, cultureCanBeInheritedFromFamily)
 	characterInput << "\tfamily=42";
 	characterInput << "}";
 
-	const Imperator::Family theFamily(familyInput, 42);
+	const auto theFamily = *Imperator::Family::Factory{}.getFamily(familyInput, 42);
```

Comment:
I've started switching to using () instead of {} for constructors, unless its directly setting member variables.

Ditto for the uses in FamilyTests